markdown/usage/nf_core_tutorial.md

```diff
@@ -13,7 +13,7 @@ subtitle: Tutorial covering the basics of using and creating nf-core pipelines.
 <iframe src="//www.slideshare.net/slideshow/embed_code/key/I9HrOeNJx1if3D" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
 <!-- markdownlint-restore -->
 
-**[Click here to download the slides associated with this tutorial.](../assets/markdown_assets/usage/nf_core_tutorial/nf-core_tutorial.pdf)**
+**[Click here to download the slides associated with this tutorial.](../../public_html/assets/markdown_assets/usage/nf_core_tutorial/nf-core_tutorial.pdf)**
```

Comment:
This one breaks: https://nf-co.re/public_html/assets/markdown_assets/usage/nf_core_tutorial/nf-core_tutorial.pdf

Better to switch it for an absolute link: https://nf-co.re/assets/markdown_assets/usage/nf_core_tutorial/nf-core_tutorial.pdf

Reply:
Yes maybe, then with the above point it may be easiest to explicitly add  https://nf-co.re/ as prefix everywhere to avoid confusion